docs/source/miniconda.rst

```diff
@@ -32,20 +32,20 @@ which does require administrator permissions.
 Latest Miniconda Installer Links
 ================================
 
-.. csv-table:: Latest - Conda 23.1.0 Python 3.10.9 released February 7, 2023
+.. csv-table:: Latest - Conda 23.3.1 Python 3.10.10 released April 24, 2023
    :header: Platform,Name,SHA256 hash
    :widths: 5, 10, 80
 
-   Windows,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe>`_,``d4517212c8ac44fd8b5ccc2d4d9f38c2dd924c77a81c2be92c3a72e70dd3e907``
+   Windows,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe>`_,``307194e1f12bbeb52b083634e89cc67db4f7980bd542254b43d3309eaf7cb358``
    ,`Miniconda3 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86.exe>`_,``4fb64e6c9c28b88beab16994bfba4829110ea3145baa60bda5344174ab65d462``
```

Comment:
Why didn't this hash change?

Reply:
We no longer build Windows 32-bit packages or installers, so the latest version we have for it is "frozen" at Miniconda v4.12.0. We still list it on this page because we _do_ have Windows 32-bit users.

Reply:
Gotcha, makes sense!